### ISSUE TITLE
alter/change styles of popover

### ISSUE THREAD

**Issue by jtc10005** [pleerock/ng2-popover#53]
is there a way to alter or change the style of the popover.  ex.  have black bgcolor w/white text?
I have tried adding my own styles like:
```
div.popover-content{
    background-color: black !important;
    color:white !important;
}
```
this has no apparent effect.

**Comment by nheinbaugh** [comment]
I noticed the same. I wasn't even able to apply styling directly to the element inline (style=....)

**Comment by PickleyD** [comment]
Try putting a wrapper div around the majority of your content and giving it an id.
For example I have a <div id="bootstrap-overrides">...</div> wrapper around mine and then I use the selector 

`#bootstrap-overrides popover-content *{
    border-radius:0px;
}`

to remove all rounded edges in my popovers.

It may also be necessary to decorate the component which imports this CSS with `encapsulation: ViewEncapsulation.None`, I'm not sure. 

Hope that helps

**Comment by nheinbaugh** [comment]
Oddly some styles (color) work while others (background-color even on !important) do not. I will have to spend some more time with it.

**Comment by jtc10005** [comment]
@PickleyD  I tried your suggestion about a wrapper with no luck.
I have now also realized that I am unable to set the width of the popover.
[http://plnkr.co/edit/StngUeJOSjwmVtN6nciL?p=preview](http://plnkr.co/edit/StngUeJOSjwmVtN6nciL?p=preview)

**Comment by jtc10005** [comment]
So after trying for a while I realized that @PickleyD  your suggestion about wrapping the ng-content in a div with an id did work.  My hmr was not reloading the styles and I did not realize that.  So thanks for that.  I also stumbled upon an alternative for anyone else here [http://stackoverflow.com/a/41090797](http://stackoverflow.com/a/41090797) using `/deep/` in styling.  I am going to go on and mark this one as closed.  Thanks for the help!
